load.php

```diff
@@ -29,6 +29,11 @@
 		'Core',
 		[
 			'enabled' => true,
+			'consent' => [
+				'display-banner'      => '',
+				'cookie-expiration'   => '',
+				'banner-options'      => '',
```

Comment:
`null` seems like a better choice for these defaults I think?

Also, shouldn't be aligned, as it causes code churn in the future.